src/SFA.DAS.Commitments.Application.UnitTests/Queries/GetActiveApprenticeshipsByUln/WhenGetApprenticeshipsByUln.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
```

Comment:
I can see there is now a new GetApprenticeshipsByUln call and this file/class has been renamed from WhenGetActiveApprenticeshipsByUln to WhenGetApprenticeshipsByUln.

Does it make sense to rename this file to remove the word 'Active', especially as there are now 2 separate calls differentiated by the word Active and this test file covers the Active version?

File renames also introduce noise into the PR (it's not easy to see any actual changes to the code in this file), so I suggest, should only be done if it adds value.

Reply:
A good spot will delete the file